### ISSUE TITLE
Error: ENOENT: no such file or directory, lstat 'out/prerendered']

### ISSUE THREAD

**Issue by cheadman** [anasmohammed361/sveltekit-aws-lambda#1]
I get the following output when trying to run `node build.js` after npm run build

 ```
node build.js
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, lstat 'out/prerendered'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'lstat',
  path: 'out/prerendered'
}

Node.js v20.5.0
```

**Comment by anasmohammed361** [comment]
The place where you put your script plays a vital role This is my folder structure , and please make sure you are running build app after you have built your sveltekit app
[
![image](https://github.com/anasmohammed361/sveltekit-aws-lambda/assets/78010612/51abe335-dfb2-4c2c-a716-91f4d31cdbc5)
](url)

And i've made  some patches on the package so please update to the latest version

**Comment by anasmohammed361** [comment]
This [Blog](https://medium.com/@anasmohammed361/sveltekit-on-aws-lambda-cloudfront-s3-79d60d572f4d) covers this adapter , I highly recommend you using SST unless you have some strong reasons not to.

**Comment by anasmohammed361** [comment]
I've patched it in `v - 1.2.3`
